domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/service/casetype/CaseTypeServiceImplTest.java

```diff
@@ -481,8 +481,10 @@ void shouldReturnFalse_whenCaseTypeForJurisdictionOtherThanGivenDoesNotExist() {
     class FindByCaseTypeIdTests {
 
         private static final String caseTypeId = "caseTypeID";
+        private final List<String> caseTypeIds = Arrays.asList(caseTypeId,caseTypeId);
         private final CaseType caseType = new CaseType();
         private final CaseTypeEntity caseTypeEntity = new CaseTypeEntity();
+        private final List<CaseTypeEntity> caseTypeEntities = Arrays.asList(caseTypeEntity,caseTypeEntity);
```

Comment:
why used same objects in list **caseTypeEntity**

Reply:
Yes because this CaseTypeRepository.findAllLatestVersions() return a list of CaseTypeEntity this iis only used for a mock.